v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 3
+title: Backend Setup
+hide_title: true
+---
+
+# Setting up the Backend
+
+In order to use the anomaly detection service, a request to the following enpoint needs to be made:
+
+```typescript
+POST https://security-us-east-1.aws.supertokens.io/v1/security
```

Comment:
Mention that this is for US region, and we have other endpoints for singapore and ireland (state them here)

Reply:
done